builders/interoperability/xcm/xc20/overview.md

```diff
@@ -141,4 +276,126 @@ The **IERC20** precompile for the XC-20 will appear in the list of **Deployed Co
 
 ![Interact with the precompile functions](/images/builders/interoperability/xcm/xc20/overview/overview-4.png)
 
-To learn how to interact with each of the functions, you can check out the [ERC-20 Precompile](/builders/pallets-precompiles/precompiles/erc20/){target=_blank} guide and modify it for interacting with the XC-20 precompile.
\ No newline at end of file
+To learn how to interact with each of the functions, you can check out the [ERC-20 Precompile](/builders/pallets-precompiles/precompiles/erc20/){target=_blank} guide and modify it for interacting with the XC-20 Precompile.
+
+## Register Moonbeam-based Assets on Another Chain {: #register-an-xc20 }
+
+In order to enable cross-chain transfers of Moonbeam native assets or ERC-20s between your chain and Moonbeam, you'll need to register the asset(s). To do so, you'll need the multilocation of each asset.
+
+There are additional steps aside from register assets that will need to be taken to enable a cross-chain integration with Moonbeam. For more information, please refer to the [Establishing an XC Integration with Moonbeam](/builders/interoperability/xcm/xc-integration){target=_blank} guide.
+
+### Register Moonbeam Native Tokens {: #moonbeam-native-tokens }
```

Comment:
I know we talked about having this here, but not sure it really makes sense 🤷‍♀️